src/net/connection/virtual_connection.rs

```diff
@@ -41,6 +53,62 @@ impl VirtualConnection {
         let now = Instant::now();
         now.duration_since(self.last_heard)
     }
+
+    /// This prepossess the given buffer to be send over the network.
```

Comment:
What do you mean by `prepossess` here? `pre-process`?

Reply:
Clion suggests it, so I thought it was spelled right like that.